src/main/java/de/rabea/Arguments.java

```diff
@@ -4,7 +4,21 @@
 
 public class Arguments {
 
-    public HashMap<String, String> parse(String[] commandLineArguments) {
+    private String[] commandLineArguments;
+
+    public Arguments(String[] commandLineArguments) {
+        this.commandLineArguments = commandLineArguments;
+    }
+
+    public int getPort() {
+        return Integer.parseInt(parseArguments().get("-p"));
```

Comment:
Any reason this can't happen once and just be stored? What is the value in storing the array of Strings?